package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-import-x",
-  "version": "0.5.3",
+  "version": "2.0.0",
```

Comment:
Manually set to `2.0.0`, so the changeset will set it to `3.0.0` in the next release.